### PR TITLE
Doc updates for release 2.11

### PR DESCRIPTION
**Pull request by anusudarsan**

### CODE REVIEW

**Comment by anusudarsan** [comment]
@mosabua ready for review

File: docs/installation/java-installation.rst [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
-=================
-Installing Java 8
-=================
+===============
+Installing Java
+===============
 Prerequisites: :ref:`presto-admin-installation-label` and :ref:`presto-admin-configuration-label`
 
-The Oracle Java 1.8 JRE (64-bit), update 45 or higher, is a prerequisite for Presto. If a suitable 64-bit version of Oracle Java 8 is already installed on the cluster, you can skip this step.
+JDK 11, is a prerequisite for Presto. If a suitable 64-bit version of JDK 11 is already installed on the cluster, you can skip this step.
```
Comment (mosabua) [inline docs/installation/java-installation.rst:8]: A Java 11 runtime is a requirement for Presto.

File: docs/installation/advanced-installation-options.rst [hunk 2]
```diff
@@ -39,10 +39,10 @@ You can switch to a new coordinator by following the steps below:
    ``~/.prestoadmin/config.json``.
 
 3. To install Presto on the new node, run the following two ``presto-admin``
-   commands. The first command is needed only if Java 8 is not already installed
+   commands. The first command is needed only if Java is not already installed
    on the new coordinator: ::
 
-     ./presto-admin package install -H new_coordinator /path/to/jdk8.rpm
+     ./presto-admin package install -H new_coordinator /path/to/jdk.rpm
```
Comment (mosabua) [inline docs/installation/advanced-installation-options.rst:45]: jre ?

File: docs/installation/java-installation.rst (hunk 1 above)
Comment (mosabua) [reply docs/installation/java-installation.rst:8]: Maybe should link to https://prestosql.io/docs/current/installation/deployment.html#requirements

**Comment by anusudarsan** [comment]
Addressed comments

**Comment by anusudarsan** [comment]
replaced with #28

**Pull request closed without merging**